www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -128,48 +103,57 @@ const Filter = (): JSX.Element => {
   return (
     <Filters
       expandLabel={labelShowCriteriasFilters}
-      expandableFilters={<Criterias />}
+      expandableFilters={
+        <Grid container item alignItems="center" spacing={1}>
+          <Criterias />
+        </Grid>
+      }
       expanded={filterExpanded}
       filters={
-        <>
-          <div className={classes.saveFilter}>
+        <Grid container item alignItems="center" spacing={1} wrap="nowrap">
+          <Grid item>
             <SaveFilter />
-          </div>
-          <Grid container alignItems="center" spacing={1}>
-            <Grid item>
-              {customFiltersLoading ? (
-                <FilterLoadingSkeleton />
-              ) : (
-                <SelectFilter
-                  ariaLabel={t(labelStateFilter)}
-                  className={classes.filterSelect}
-                  options={options.map(pick(['id', 'name', 'type']))}
-                  selectedOptionId={canDisplaySelectedFilter ? filter.id : ''}
-                  onChange={changeFilter}
-                />
-              )}
-            </Grid>
-            <Grid item>
-              <SearchField
-                EndAdornment={SearchHelpTooltip}
-                placeholder={t(labelSearch)}
-                value={nextSearch || ''}
-                onChange={prepareSearch}
-                onKeyDown={requestSearchOnEnterKey}
+          </Grid>
+          <Grid item>
+            {customFiltersLoading ? (
+              <FilterLoadingSkeleton />
+            ) : (
+              <SelectFilter
+                ariaLabel={t(labelStateFilter)}
+                options={options.map(pick(['id', 'name', 'type']))}
+                selectedOptionId={canDisplaySelectedFilter ? filter.id : ''}
+                onChange={changeFilter}
               />
-            </Grid>
+            )}
+          </Grid>
+          {filterExpanded ? (
+            <>
+              <Grid item>
+                <SearchField
+                  EndAdornment={SearchHelpTooltip}
+                  placeholder={t(labelSearch)}
+                  value={nextSearch || ''}
+                  onChange={prepareSearch}
+                  onKeyDown={requestSearchOnEnterKey}
+                />
+              </Grid>
+              <Grid item>
+                <Button
+                  color="primary"
+                  size="small"
+                  variant="text"
```

Comment:
Why the button is not contained anymore ?

Reply:
> Why the button is not contained anymore ?

Fixed